src/Compilers/CSharp/Portable/Binder/Semantics/Operators/BinaryOperatorOverloadResolution.cs

```diff
@@ -825,25 +834,95 @@ private static LiftingResult UserDefinedBinaryOperatorCanBeLifted(TypeSymbol lef
                     {
                         continue;
                     }
+
                     var better = BetterOperator(candidates[i].Signature, candidates[j].Signature, left, right, ref useSiteDiagnostics);
-                    if (better == BetterResult.Left)
+                    if (better == BetterResult.Left && candidates[j].Kind != OperatorAnalysisResultKind.Worse)
```

Comment:
Success is the most common scenario, we are, usually, willing to pay a little bit more for error cases. Preserving the original behavior is one of our goals here, I believe.